envoy/ssl/context.h

```diff
@@ -13,13 +13,23 @@ namespace Ssl {
 
 using CertificateDetailsPtr = std::unique_ptr<envoy::admin::v3::CertificateDetails>;
 
+struct CertificateNameExpiration {
+  std::string name;
+  uint64_t expiration;
```

Comment:
This interface would be nicer if it held a `std::chrono::duration`

Reply:
Sounds good. Updated